src/core/Menu/Menu.test.tsx

```diff
@@ -17,9 +17,9 @@ const TestMenuLanguage = (
   </Menu.language>
 );
 
-test('CSS export', () => {
-  const css = cssFromBaseStyles(baseStyles);
-  expect(css).toEqual(expect.stringContaining('data-reach-menu-button'));
-});
+// test('CSS export', () => {
+//   const css = cssFromBaseStyles(baseStyles);
+//   expect(css).toEqual(expect.stringContaining('data-reach-menu-button'));
+// });
```

Comment:
These commented-out codes and imports might have left by mistake. Consider removing these.